### PR TITLE
Update getting-started-for-flutter.phtml

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- mention minimum macOS version

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

- https://github.com/appwrite/sdk-for-flutter/issues/147

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

File: app/views/docs/getting-started-for-flutter.phtml [hunk 1]
```diff
@@ -70,6 +70,7 @@ $version = (isset($versions['flutter'])) ? $versions['flutter'] : '';
 <h3><a href="/docs/getting-started-for-flutter#macOS" id="macOS">Mac OS</a></h3>
 
 <p>For <b>Mac OS</b> add your app name and Bundle ID, You can find your Bundle Identifier in the General tab for your app's primary target in Xcode.</p>
+<p>The Appwrite SDK uses ASWebAuthenticationSession on macOS 10.15+ to allow OAuth authentication. You have to change your macOS Deployment Target in Xcode to be macOS >= 10.15 to be able to build your app for macOS.</p>
```
Comment (gewenyu99) [inline app/views/docs/getting-started-for-flutter.phtml:73]: Do we want to use `MacOS` `Mac OS` or `macOS`? What's the official format? let's pick one and stick to it.

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

File: app/views/docs/getting-started-for-flutter.phtml (hunk 1 above)
Comment (lohanidamodar) [reply app/views/docs/getting-started-for-flutter.phtml:73]: image
Its macOS

**Review by stnguyen90 — APPROVED** [review]

File: app/views/docs/getting-started-for-flutter.phtml (hunk 1 above)
Comment (gewenyu99) [reply app/views/docs/getting-started-for-flutter.phtml:73]: Okay，let's update all occurences of Mac OS and MacOS to macOS. Just so we're consistent.

**Review by gewenyu99 — APPROVED** [review]
I agree with all of this :D

**Review by eldadfux — APPROVED** [review]

**Pull request merged**